### ISSUE TITLE
[Broken Website]  Gadgety website sometimes have its photos getting hidden

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [darkreader/darkreader#3643]
**Website**
The link of the broken website
https://www.gadgety.co.il/258305/lg-wing-announced/

**How to Reproduce**
An example of this is
> 1. Go to the website, and compare between light and dark theme. 

**Expected behavior**
Images should be visible.

**Actual behavior**
You can see that on dark theme, the images might get hidden

**Screenshots**
![image](https://user-images.githubusercontent.com/5357526/93117637-d0c94e80-f6c7-11ea-90d2-83c461a680bf.png)

![image](https://user-images.githubusercontent.com/5357526/93117769-00785680-f6c8-11ea-9417-38e9265a5ccf.png)

**System info:**
 - OS: [e.g. Windows, MacOS, Linux]
Windows 10
 - Browser: [e.g. chrome, safari]
Chromium based 80.0.3987.163 (Cent Browser 4.2.10.171)
 - Darkreader Version: [e.g. 4.9.9]
4.9.19

 
**Additional context**
Would be nice if there was a setting to avoid modifying how images look .

**Comment by Gusted** [comment]
Hi @AndroidDeveloperLB!

The mode your using in Static and that isn't a ready to go style and should be modified itself for certain sites to make it look good.
If you want to have a quick fix use Filter(+) or Dynamic mode.



> Would be nice if there was a setting to avoid modifying how images look .

Fortunate we don't change any images it's rather the images is getting blocked by certain elements(their background-color).

Regards,
Gusted

**Comment by AndroidDeveloperLB** [comment]
Oh so it changes only the background?
But then how come I don't see the content of the images?

**Comment by Gusted** [comment]
> blocked by certain elements(their background-color).

Well, it seems like their are certain elements that lay on top of the image(z-index), that is filling that area with it's background-color.

**Comment by AndroidDeveloperLB** [comment]
Is it possible to choose only the main background instead?

**Comment by Gusted** [comment]
> Is it possible to choose only the main background instead?

Than you should edit the default stylesheet to specify this trough the devtools.

```CSS
/* Common theme */
html,
body,
:not([style*="background-color:"]) {
    background-color: rgb(16, 20, 23) !important;
}
html,
body,
:not([style*="color:"]) {
    color: rgb(167, 158, 139) !important;
}
h1:not([style*="color:"]),
h2:not([style*="color:"]),
h3:not([style*="color:"]),
h4:not([style*="color:"]),
h5:not([style*="color:"]),
h6:not([style*="color:"]) {
    color: rgb(247, 142, 102) !important;
}
cite:not([style*="color:"]) {
    color: rgb(128, 204, 148) !important;
}
input:not([style*="background-color:"]),
textarea:not([style*="background-color:"]),
button:not([style*="background-color:"]),
[role="button"] {
    background-color: rgb(32, 48, 64) !important;
}
input:not([style*="background-color:"]):hover,
textarea:not([style*="background-color:"]):hover,
button:not([style*="background-color:"]):hover,
[role="button"]:hover {
    background-color: rgb(30, 54, 78) !important;
}
input:not([style*="background-color:"]):active, input:not([style*="background-color:"]):focus,
textarea:not([style*="background-color:"]):active, textarea:not([style*="background-color:"]):focus,
button:not([style*="background-color:"]):active, button:not([style*="background-color:"]):focus,
[role="button"]:active, [role="button"]:focus {
    background-color: rgb(29, 56, 83) !important;
}
a:not([style*="color:"]) {
    color: rgb(128, 182, 204) !important;
}
a:not([style*="color:"]):hover {
    color: rgb(142, 193, 217) !important;
}
a:not([style*="color:"]):active, a:not([style*="color:"]):focus {
    color: rgb(155, 203, 230) !important;
}
:not([style*="border-color:"]),
::before,
::after {
    border-color: rgb(80, 115, 134) !important;
}
div:empty {
    background-color: rgb(16, 20, 23) !important;
}
input::placeholder,
textarea::placeholder {
    color: rgb(167, 158, 139) !important;
}
input:not([style*="background-image:"]),
textarea:not([style*="background-image:"]) {
    background-image: none !important;
}
```

**Comment by AndroidDeveloperLB** [comment]
That's quite a lot...
Can you please consider adding it as a feature?

**Comment by Gusted** [comment]
> Can you please consider adding it as a feature?

What feature? Static isn't made to be out of the box. It's rather just a static Stylesheet as fundation to build other CSS on it. E.g. a generated Dynamic CSS.

**Comment by AndroidDeveloperLB** [comment]
Sorry for upsetting you. I'm not an expert in HTML, and not in this addon either.
The feature I asked is whether we could have, like the other customizations, a checkbox to choose whether to set the background to all elements as it's now, or just the main one, of the page.

**Comment by Gusted** [comment]
> Sorry for upsetting you. I'm not an expert in HTML, and not in this addon either.

Than like I said, Static isn't for the normal user and shouldn't be even from my opinion used out of the box.
If you search for customization, you should be looking for Dynamic as we [recently added those](https://darkreader.org/blog/v4-9/)

**Comment by AndroidDeveloperLB** [comment]
Isn't "static" the default one?
It works nicely for most websites...

**Comment by Gusted** [comment]
> Isn't "static" the default one?
> It works nicely for most websites...

No, Dynamic is actually the default one as it give the best results.

**Comment by AndroidDeveloperLB** [comment]
@Gusted I must have changed it, then.
Do you recommend using "Dynamic", and change it for specific websites?

**Comment by Gusted** [comment]
> Do you recommend using "Dynamic", and change it for specific websites?

Yes, as it will analyze the site's stylesheet and elements and make according colors for it.

> Are there performance-related concerns too?

If you have possible concerns I recommend Filter+

**Comment by AndroidDeveloperLB** [comment]
@Gusted OK I think I know what happened.
Dynamic is indeed the default, but for this website I chose "static", because using "dynamic" ruins the comments section, as the text becomes black on black background.
So, it's either hidden images or almost-hidden comments .

:(

Choosing  Filter+ seems to handle both. Hopefully I won't notice other issues.

**Comment by Gusted** [comment]
> Dynamic is indeed the default, but for this website I chose "static", because using "dynamic" ruins the comments section, as the text becomes black on black background.
> So, it's either hidden images or almost-hidden comments .

But as opposed to static mode. Fixes for dynamic/Filter(+) mode can be made remotely
e.g. 
https://github.com/darkreader/darkreader/blob/master/src/config/dynamic-theme-fixes.config

**Comment by AndroidDeveloperLB** [comment]
@Gusted That's way more than what I know.
Sorry.
It's not an easy solution.

**Comment by AndroidDeveloperLB** [comment]
@Gusted When using "filter+", I've noticed it sometimes affects YouTube videos. See:
https://www.gadgety.co.il/259483/free-guy-trailer/

![image](https://user-images.githubusercontent.com/5357526/95175675-7b62f780-07c4-11eb-839f-c169d84c51c0.png)

Why would the filtering affect videos of YouTube?

**Comment by AndroidDeveloperLB** [comment]
I also tried to copy the text you wrote for "static" mode, and when I hover the mouse over YouTube videos while playing, I see blank, dark content:

![image](https://user-images.githubusercontent.com/5357526/95175940-d694ea00-07c4-11eb-84ab-814b52174fad.png)

**Comment by Gusted** [comment]
> Why would the filtering affect videos of YouTube?

Because it's bad targeting the youtube video's, it's being fixed by #3793

> I also tried to copy the text you wrote for "static" mode, and when I hover the mouse over YouTube videos while playing, I see blank, dark content:

Hmm weird...

**Comment by AndroidDeveloperLB** [comment]
@Gusted Can you please check it out?
